documentation/configuringtheplugin.md

```diff
@@ -0,0 +1,28 @@
+# Configuring the Synopsys Detect Plugin
+Use the following process to configure the Synopsys Detect for Jenkins plugin.  Note that the supported credentials formats are user name and password or API token.  SAML is not supported.
+
```

Comment:
We have put names similar to plugin names into keywords.ditamap... that could be something to consider.

There's an existing keyword for "Black Duck" that could be used below.

Reply:
Well this is a bit confusing... you should already see the use of existing keywords amongst other changes in that file.
re: https://github.com/blackducksoftware/synopsys-detect/blob/cp_doc_Adding_GitLab/documentation/src/main/markdown/integrations/jenkinsplugin/configuringtheplugin.md?plain=1